### ISSUE TITLE
Deprecation warning - Python 3.9

### ISSUE THREAD

**Issue by hsarji** [elapouya/python-docx-template#358]
I'm not sure why I'm getting the following message: 

```
Documents/site-packages/docx/section.py:7: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working                                                                                                   
  from collections import Sequence            
```

**Comment by elapouya** [comment]
python-docx-template uses python-docx : it seems we will have a problem with python 3.9 if its authors do not make changes...

**Comment by hsarji** [comment]
I just looked at the issues for python-docs and [there seems to be a fix](https://github.com/python-openxml/python-docx/issues/927#issuecomment-841735013). But if the fix was released on May 15, 2021, why would I still be seeing the deprecation warning?

**Comment by elapouya** [comment]
python-docx-template takes the latest version of python-docx on install : I guess you install them before May 15. Just upgrade python-docx, it should be enough.

**Comment by hsarji** [comment]
Everything seems up-to-date, yet I'm still getting the error. 
![IMG_1736](https://user-images.githubusercontent.com/57333897/119965817-af1fa700-bf78-11eb-8e0a-f583fcc863c2.png)
![IMG_1737](https://user-images.githubusercontent.com/57333897/119965945-ceb6cf80-bf78-11eb-8079-c8488792969d.png)
![IMG_1738](https://user-images.githubusercontent.com/57333897/119965829-b21a9780-bf78-11eb-9b94-b9514f8320dd.png)

**Comment by elapouya** [comment]
what gives `pip list` ?

**Comment by hsarji** [comment]
I'm not sure. Where are you seeing `pip list`? I'm new to Python and learning things as I go. I'm trying to automate documents from iPad. I don't want my efforts to go to waste if things get deprecated soon.

**Comment by hsarji** [comment]
Thank you for suggesting `pip list`. I did some research to learn what you meant. Apparently, despite my earlier screen captures, Pyto wasn't using the latest version of python-docx. As the following screen capture shows, it was using version 0.8.10, instead of 0.8.11, which has the fix:
![IMG_1740](https://user-images.githubusercontent.com/57333897/119988614-48a88200-bf94-11eb-968d-2bdd04d9cc44.jpeg)
I followed the suggestion made by Pyto (see the next screen capture). I deleted python-docx and installed it again. I'm no longer getting the error message! 
![IMG_1742](https://user-images.githubusercontent.com/57333897/119988687-5b22bb80-bf94-11eb-8e4e-10beb06251af.jpeg)
Thank you for your help!
